contracts/test/nativeTransfer.mjs

```diff
@@ -0,0 +1,182 @@
+import hardhat from "hardhat"
+import chai from "chai"
+import {getGasPrice, getTxReceipt} from "../utils/tx.js"
+
+import {isGwMainnetV1} from "../utils/network.js"
+
+const {ethers} = hardhat
+const {expect} = chai
+const {BigNumber} = ethers
+
+let gasPrice, EOA0, EOA1, newEOA0, CA0
+
+if (!isGwMainnetV1()) {
+    gasPrice = await getGasPrice(ethers.provider)
+    EOA0 = (await ethers.getSigners())[0].address
+    EOA1 = (await ethers.getSigners())[1].address
+    newEOA0 = ethers.Wallet.createRandom().address
+    let baseFallbackReceive = await ethers.getContractFactory("baseFallbackReceive")
+    let contract = await baseFallbackReceive.deploy()
+    await contract.deployed()
+    CA0 = contract.address
+}
+
+describe("transfer success", function () {
+    if (isGwMainnetV1()) {
+        return;
+    }
+
+    let tests = [
+        {name: "to EOA", from: EOA0, to: EOA1, value: "0x1", expectGasUsed: "21000"},
+        {name: "to EOA tx.data is not null", from: EOA0, to: EOA1, value: "0x1", data: "0x12", expectGasUsed: "21016"},
+        {name: "to itself", from: EOA0, to: EOA0, value: "0x10", expectGasUsed: "21000"},
+        {name: "transfer 0", from: EOA0, to: EOA1, value: "0x0", expectGasUsed: "21000"},
+        {name: "transfer big value", from: EOA0, to: EOA1, value: "0x8ac7230489e80000", expectGasUsed: "21000"},
+        {name: "to new EOA", from: EOA0, to: newEOA0, value: "0x100", expectGasUsed: "46000"},
+        {name: "to CA", from: EOA0, to: CA0, value: "0x200", expectGasUsed: "21033"},
+        {name: "to CA tx.data is not null", from: EOA0, to: CA0, data: "0x12", value: "0x300", expectGasUsed: "21050"},
+    ]
+
+    for (let i = 0; i < tests.length; i++) {
+        let test = tests[i]
+        it(test.name, async () => {
+            let from_balance = await ethers.provider.getBalance(test.from)
```

Comment:
For variables that will not be modified, we can use `const` instead of `let` to define it, which is easier to understand.
```suggestion
            const from_balance = await ethers.provider.getBalance(test.from)
```

Reply:
Okay, I have modified it.